Sources/Playbook/Components/Highlight/docs/_highlight_default_swift.md

```diff
@@ -0,0 +1,8 @@
+```swift
+VStack(alignment: .leading, spacing: Spacing.small) {
+      PBHighlight(
```

Comment:
I think the indentation looks off here

Reply:
I changed the indentation idk if it's better or not. I did control + i in Xcode and that was the indentation it used(in the first example that you commented on) so just let me know if its still off!

Reply:
Yes. It's still off. I think ctrl+i doesn't work with .md files

Reply:
Our code convention is to use 2 spaces.